cni/network/network.go

```diff
@@ -460,6 +494,14 @@ func (plugin *NetPlugin) Add(args *cniSkel.CmdArgs) error {
 				zap.Any("results", ipamAddResult))
 			return plugin.Errorf(errMsg)
 		}
+	} else if !nwCfg.MultiTenancy && nwCfg.IPAM.Type == network.AzureCNS {
```

Comment:
wanted to understand why we need this change

Reply:
because for swiftv2 path, the nwconfig multitenancy must not be set to true which was confused before; and the nwcfg ipam type should be azureCNS